demos/python_demos/colorization/colorization_demo.py

```diff
@@ -0,0 +1,106 @@
+from openvino.inference_engine import IENetwork, IECore
+import cv2 as cv
+import numpy as np
+import os
+from argparse import ArgumentParser, SUPPRESS
+import xml.etree.ElementTree as ET
+import re
+import logging as log
+import glob
+import sys
+
+
+def build_arg():
+    parser = ArgumentParser(add_help=False)
+    in_args = parser.add_argument_group('Options')
+    in_args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Help with the script.')
+    in_args.add_argument("-m", "--model", help="Required. Path to .xml file with pre-trained model",
+                         required=True, type=str)
+    in_args.add_argument("-d", "--device",
+                         help="Optional. Specify target device for infer: CPU, GPU, FPGA, HDDL or MYRIAD. "
+                         "Default: CPU",
+                         default="CPU", type=str)
+    in_args.add_argument("-i", "--input", help="Required. Path to a folder with image files", required=True, type=str)
```

Comment:
Please, add `-no_show` key to simplify future testing.

Reply:
@Wovchena done